src/android/FirebaseAuthenticationPlugin.java

```diff
@@ -246,4 +248,49 @@ private static PluginResult getProfileResult(FirebaseUser user) {
             return new PluginResult(PluginResult.Status.ERROR, e.getMessage());
         }
     }
+
+    /**
+     * Updates the current user's profile data.
```

Comment:
Lets be consistent and remove javadocs from Java code.

Reply:
Removed javadocs